storybook/qmlTests/tests/tst_SwapModal.qml

```diff
@@ -1815,5 +1815,51 @@ Item {
                 compare(amountToSendInput.input.text, expectedAmount)
             }
         }
+
+        function test_no_auto_refresh_when_proposalLoading_or_approvalPending() {
+            fetchSuggestedRoutesCalled.clear()
+            root.swapFormData.fromTokenAmount = "0.0001"
+            root.swapFormData.selectedAccountAddress = "0x7F47C2e18a4BBf5487E6fb082eC2D9Ab0E6d7240"
+            root.swapFormData.selectedNetworkChainId = 11155111
+            root.swapFormData.fromTokensKey = "ETH"
+            // for testing making it 1.2 seconds so as to not make tests running too long
+            root.swapFormData.autoRefreshTime = 1200
+
+            // Launch popup
+            launchAndVerfyModal()
+
+            // check if fetchSuggestedRoutes called
+            fetchSuggestedRoutesCalled.wait()
+            compare(fetchSuggestedRoutesCalled.count, 1)
```

Comment:
I don't get the necessity to call `wait()` on the SignalSpy; it should be possible to just do:
```qml
tryCompare(fetchSuggestedRoutesCalled, "count", 1)
```

Reply:
Same below (and everywhere) in fact; can you pls try to remove the `wait()` calls?

Reply:
here I can definitely remove it!